### DISCUSSION TITLE
Multiple Analyser Carrier Board

### DISCUSSION THREAD

**Discussion by ashlin4010** [gusmanb/logicanalyzer#177] [Ideas]
So the main feature I value is your design's high channel capacity.

I would appreciate it if you were to design a carrier PCB that I could mount 3 or more analysers on in a compact and convenient way. Potentially add some pads so a users could use just one switch.

Or just provide KiCad footprint.

**Comment by gusmanb** [comment]
You don't need a carrier, the most compact way to assemble the boards is stacking them. If you want to use a single switch you can put one in one of the boards and wire the others in parallel, it should work but I would recommend to use multiple switches as it will be more flexible (you can trigger a capture from a single event and capture from different IOs with different voltages, for example an FPGA with 3.3v IOs and 1.6v IOs).

Check this post for more info on stacking: https://github.com/gusmanb/logicanalyzer/discussions/157
